ramalama/common.py

```diff
@@ -55,14 +57,45 @@ def container_manager():
     return _engine
 
 
-def is_podman_machine_running_with_krunkit():
-    podman_machine_list = ["podman", "machine", "list", "--all-providers"]
+def apple_vm():
+    podman_machine_list = ["podman", "machine", "list", "--format", "json", "--all-providers"]
```

Comment:
json is a great idea, the existing parsing was more hacky. Although I would have kept the loose *krun* check, the name there has changed before between libkrun and krunkit.